### ISSUE TITLE
Inconsistent tab order

### ISSUE THREAD

**Issue by eyalroz** [YoavHaifa/WorkersRightsComputer#32]
The tab order among controls should correspond to the order of their placement  on the windows/dialogs; right now, it doesn't - at least in the main window.

So, either the tab order should be changed or the item placement should.

**Comment by eyalroz** [comment]
Yoav, I would like to ask you to work on this, using our "monthly couplon" scheme.
